Exchange/SelectCurrencyView/SelectCurrencyViewModel.swift

```diff
@@ -76,6 +76,7 @@ extension SelectCurrencyViewModel: ViewModelType {
             .emit(onNext: router.showAlert)
         
         let isLoading = viewModels
+            .startWith([])
             .asDriver()
             .map { $0.isEmpty }
```

Comment:
Допустимо, но вот так естественнее читается вроде. На твое усмотрение
```suggestion
            .asDriver()
            .map { $0.isEmpty }
            .startWith(true)
```

Reply:
Исправил, согласен, что если нам в итоге нужно булево значение то нет смысла преобразование делать